app/routes/events/components/EventDetail/index.tsx

```diff
@@ -426,11 +426,20 @@ export default class EventDetail extends Component<Props, State> {
               iconName="time-outline"
               content={<FromToTime from={event.startTime} to={event.endTime} />}
             />
+            {event.isForeignLanguage !== null && event.isForeignLanguage && (
+              <TextWithIcon
+                iconName="language-outline"
+                content={event.isForeignLanguage ? 'English' : 'Norsk'}
```

Comment:
Since `event.isForeignLanguage` will always be `true` here, you may just write
```suggestion
                content="English"
```